### ISSUE TITLE
Error reqest playlist youtube

### ISSUE THREAD

**Issue by sangnekk** [lavalink-devs/youtube-source#96]
### Basic Troubleshooting

- [X] I have checked for similar issues (open AND closed), and my issue is not a duplicate. (MANDATORY)
- [X] I have checked for pull requests that may already address my issue. (MANDATORY)
- [X] I am using (and my issue is reproducible on) the latest version of youtube-source.
- [X] My issue is reproducible with IPv6 rotation.

### Version of youtube-source

1.11.2

### The search query/queries, URL(s), playlist ID(s)/URL(s) or video ID(s)/URL(s) that triggered the issue

```
-9] l.server.player.AudioLoaderRestHandler   : Got request to load for identifier "https://youtube.com/playlist?list=RDxnIIoan7ft8&playnext=1&si=bqpHTlPqvW3o4rCJ" 2024-12-26T21:52:12.716+07:00 ERROR 2480 --- [Lavalink] [  XNIO-1 task-9] lavalink.server.player.AudioLoader       : Load failed
```

### Code Example

```yaml
server:
  port: 2333
  address: 0.0.0.0
lavalink:
  server:
    sources:
      youtube: false # disable youtube on lavalink to use youtube-source
    password: "localhost"
    bufferDurationMs: 225
    frameBufferDurationMs: 5000
    youtubePlaylistLoadLimit: 3
    opusEncodingQuality: 5
    resamplingQuality: MEDIUM
    trackStuckThresholdMs: 5000
    playerUpdateInterval: 3
    useSeekGhosting: true
    youtubeSearchEnabled: true
    soundcloudSearchEnabled: true
    gc-warnings: true
  plugins:
    - dependency: "com.github.topi314.lavasrc:lavasrc-plugin:4.3.0" # LavaSrc - extra sources like deezer, spotify, etc. https://github.com/topi314/LavaSrc
      repository: "https://maven.lavalink.dev/releases"
    - dependency: "com.github.topi314.lavasearch:lavasearch-plugin:1.0.0" # https://github.com/topi314/LavaSearch
      repository: "https://maven.lavalink.dev/releases"
    - dependency: "com.dunctebot:skybot-lavalink-plugin:1.7.0" # https://github.com/DuncteBot/skybot-lavalink-plugin
      repository: "https://maven.lavalink.dev/releases"
    - dependency: "com.github.devoxin:lavadspx-plugin:0.0.5" # https://github.com/devoxin/LavaDSPX-Plugin
      repository: "https://jitpack.io"
    - dependency: "dev.lavalink.youtube:youtube-plugin:1.11.1" # https://github.com/lavalink-devs/youtube-source
      snapshot: false
    - dependency: "com.github.topi314.lavalyrics:lavalyrics-plugin:1.0.0" # https://github.com/topi314/LavaLyrics
      repository: "https://maven.lavalink.dev/releases"
    - dependency: "me.duncte123:java-lyrics-plugin:1.6.5" # https://github.com/DuncteBot/java-timed-lyrics
      repository: "https://maven.lavalink.dev/releases"
plugins:
  lyrics:
    countryCode: de
    geniusApiKey: "<insert>"
  lavalyrics:
    sources:
      - genius
      - spotify
      - youtube
      - deezer
      - yandexMusic
  youtube:
    enabled: true
    oauth:
      enabled: true
    allowSearch: true
    allowDirectVideoIds: true
    allowDirectPlaylistIds: true
    clients:
      - "ANDROID_MUSIC"
      - "MUSIC"
      - "WEB"
      - "WEBEMBEDDED"
      - "TVHTML5EMBEDDED"
    ANDROID_MUSIC:
      playlistLoading: false
      videoLoading: true
      searching: true
      playback: true
    MUSIC:
      playlistLoading: false
      videoLoading: true
      searching: true
      playback: false
    WEB:
      playlistLoading: false
      videoLoading: true
      searching: true
      playback: true
    WEBEMBEDDED:
      playlistLoading: false
      videoLoading: false
      searching: false
      playback: true
    TVHTML5EMBEDDED:
      playlistLoading: false
      videoLoading: false
      searching: false
      playback: true
  lavasrc:
    providers:
      - "ytsearch:\"%ISRC%\""
      - "dzisrc:%ISRC%"
      - "ytsearch:%QUERY%"
      - "dzsearch:%QUERY%"
      - "scsearch:%QUERY%"
    sources:
      spotify: true
      applemusic: false
      deezer: false
      yandexmusic: false
      flowerytts: false
      youtube: true
    lyrics-sources:
      spotify: true # Enable Spotify lyrics source
      deezer: true # Enable Deezer lyrics source
      youtube: true # Enable YouTube lyrics source
      yandexMusic: true
    spotify:
      clientId: ""
      clientSecret: ""
      spDc: "<insert>"
      countryCode: "VN"
      playlistLoadLimit: 50 # The number of pages at 100 tracks each
      albumLoadLimit: 25 # The number of pages at 50 tracks each
      localFiles: true
    applemusic:
      countryCode: "VN"
      mediaAPIToken: "<insert>"
      playlistLoadLimit: 10 # The number of pages at 300 tracks each
      albumLoadLimit: 10 # The number of pages at 300 tracks each
    deezer:
      masterDecryptionKey: "<insert>"
    youtube:
      countryCode: "VN"
      playlistLoadLimit: 50 # The number of pages at 100 tracks each
      albumLoadLimit: 25 # The number of pages at 50 tracks each
      artistLoadLimit: 5 # The number of pages at 10 tracks each
    yandexmusic:
      accessToken: "<insert>"
    flowerytts:
      voice: "Olivia" # (case-sensitive) get default voice from here https://api.flowery.pw/v1/tts/voices
      translate: false # whether to translate the text to the native language of voice
      silence: 0 # the silence parameter is in milliseconds. Range is 0 to 10000. The default is 0.
      speed: 1.0 # the speed parameter is a float between 0.5 and 10. The default is 1.0. (0.5 is half speed, 2.0 is double speed, etc.)
      audioFormat: "mp3" # supported formats are: mp3, ogg_opus, ogg_vorbis, aac, wav, and flac. Default format is mp3
  dunctebot:
      ttsLanguage: "en-US"
      sources:
        getyarn: true # www.getyarn.io
        clypit: true # www.clyp.it
        tts: true # speak:Words to speak
        pornhub: true # should be self-explanatory
        reddit: true # should be self-explanatory
        ocremix: true # www.ocremix.org
        tiktok: true # tiktok.com
        mixcloud: true # mixcloud.com
        soundgasm: true # soundgasm.net
metrics:
  prometheus:
    enabled: false
    endpoint: /metrics
sentry:
  dsn: ""
  environment: ""
logging:
  file:
    max-history: 5
    max-size: 10MB
  path: ./logs/
  level:
    root: INFO
    lavalink: INFO
  request:
    enabled: true
    includeClientInfo: true
    includeHeaders: true
    includeQueryString: true
    includePayload: true
    maxPayloadLength: 10000
  logback:
    rollingpolicy:
      max-file-size: 10MB
      max-history: 5
```
### Exception and Stacktrace
```
n:"localhost", Accept:"*/*", Connection:"keep-alive", User-Agent:"Bun/1.1.38", Host:"103.245.236.105:2333", Accept-Encoding:"gzip, deflate, br", Content-Length:"0"]
2024-12-26T21:52:11.904+07:00  INFO 2480 --- [Lavalink] [  XNIO-1 task-9] l.server.player.AudioLoaderRestHandler   : Got request to load for identifier "https://youtube.com/playlist?list=RDxnIIoan7ft8&playnext=1&si=bqpHTlPqvW3o4rCJ"
2024-12-26T21:52:12.716+07:00 ERROR 2480 --- [Lavalink] [  XNIO-1 task-9] lavalink.server.player.AudioLoader       : Load failed

com.sedmelluq.discord.lavaplayer.tools.FriendlyException: This client cannot load playlists
        at dev.lavalink.youtube.clients.TvHtml5Embedded.loadPlaylist(TvHtml5Embedded.java:124) ~[jar:file:C:\Users\Administrator\Desktop\Music Port\.\plugins\youtube-plugin-1.11.1.jar!/:na]
        at dev.lavalink.youtube.YoutubeAudioSourceManager.lambda$getRouter$2(YoutubeAudioSourceManager.java:277) ~[jar:file:C:\Users\Administrator\Desktop\Music Port\.\plugins\youtube-plugin-1.11.1.jar!/:na]
        at dev.lavalink.youtube.YoutubeAudioSourceManager.loadItemOnce(YoutubeAudioSourceManager.java:218) ~[jar:file:C:\Users\Administrator\Desktop\Music Port\.\plugins\youtube-plugin-1.11.1.jar!/:na]
        at dev.lavalink.youtube.YoutubeAudioSourceManager.loadItem(YoutubeAudioSourceManager.java:183) ~[jar:file:C:\Users\Administrator\Desktop\Music Port\.\plugins\youtube-plugin-1.11.1.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.checkSourcesForItemOnce(DefaultAudioPlayerManager.java:442) ~[lavaplayer-2.2.1.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.checkSourcesForItem(DefaultAudioPlayerManager.java:423) ~[lavaplayer-2.2.1.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.loadItemSync(DefaultAudioPlayerManager.java:154) ~[lavaplayer-2.2.1.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.loadItemSync(DefaultAudioPlayerManager.java:167) ~[lavaplayer-2.2.1.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.player.AudioPlayerManager.loadItemSync(AudioPlayerManager.java:103) ~[lavaplayer-2.2.1.jar!/:na]
        at lavalink.server.player.AudioLoader.load(AudioLoader.kt:56) ~[classes!/:4.0.7]
        at lavalink.server.player.AudioLoaderRestHandler.loadTracks(AudioLoaderRestHandler.kt:56) ~[classes!/:4.0.7]
        at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103) ~[na:na]
        at java.base/java.lang.reflect.Method.invoke(Method.java:580) ~[na:na]
        at kotlin.reflect.jvm.internal.calls.CallerImpl$Method.callMethod(CallerImpl.kt:97) ~[kotlin-reflect-2.0.0.jar!/:2.0.0-release-341]
        at kotlin.reflect.jvm.internal.calls.CallerImpl$Method$Instance.call(CallerImpl.kt:113) ~[kotlin-reflect-2.0.0.jar!/:2.0.0-release-341]
        at kotlin.reflect.jvm.internal.KCallableImpl.callDefaultMethod$kotlin_reflection(KCallableImpl.kt:207) ~[kotlin-reflect-2.0.0.jar!/:2.0.0-release-341]
        at kotlin.reflect.jvm.internal.KCallableImpl.callBy(KCallableImpl.kt:112) ~[kotlin-reflect-2.0.0.jar!/:2.0.0-release-341]
        at org.springframework.web.method.support.InvocableHandlerMethod$KotlinDelegate.invokeFunction(InvocableHandlerMethod.java:334) ~[spring-web-6.1.9.jar!/:6.1.9]
        at org.springframework.web.method.support.InvocableHandlerMethod.doInvoke(InvocableHandlerMethod.java:252) ~[spring-web-6.1.9.jar!/:6.1.9]
        at org.springframework.web.method.support.InvocableHandlerMethod.invokeForRequest(InvocableHandlerMethod.java:188) ~[spring-web-6.1.9.jar!/:6.1.9]
        at org.springframework.web.servlet.mvc.method.annotation.ServletInvocableHandlerMethod.invokeAndHandle(ServletInvocableHandlerMethod.java:118) ~[spring-webmvc-6.1.8.jar!/:6.1.8]
        at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.invokeHandlerMethod(RequestMappingHandlerAdapter.java:926) ~[spring-webmvc-6.1.8.jar!/:6.1.8]
        at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.handleInternal(RequestMappingHandlerAdapter.java:831) ~[spring-webmvc-6.1.8.jar!/:6.1.8]
        at org.springframework.web.servlet.mvc.method.AbstractHandlerMethodAdapter.handle(AbstractHandlerMethodAdapter.java:87) ~[spring-webmvc-6.1.8.jar!/:6.1.8]
        at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:1089) ~[spring-webmvc-6.1.8.jar!/:6.1.8]
        at org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:979) ~[spring-webmvc-6.1.8.jar!/:6.1.8]
        at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:1014) ~[spring-webmvc-6.1.8.jar!/:6.1.8]
        at org.springframework.web.servlet.FrameworkServlet.doGet(FrameworkServlet.java:903) ~[spring-webmvc-6.1.8.jar!/:6.1.8]
        at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:527) ~[jakarta.servlet-api-6.0.0.jar!/:6.0.0]
        at org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:885) ~[spring-webmvc-6.1.8.jar!/:6.1.8]
        at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:614) ~[jakarta.servlet-api-6.0.0.jar!/:6.0.0]
        at io.undertow.servlet.handlers.ServletHandler.handleRequest(ServletHandler.java:74) ~[undertow-servlet-2.3.13.Final.jar!/:2.3.13.Final]
        at io.undertow.servlet.handlers.FilterHandler$FilterChainImpl.doFilter(FilterHandler.java:129) ~[undertow-servlet-2.3.13.Final.jar!/:2.3.13.Final]
        at org.springframework.web.filter.AbstractRequestLoggingFilter.doFilterInternal(AbstractRequestLoggingFilter.java:289) ~[spring-web-6.1.9.jar!/:6.1.9]
        at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116) ~[spring-web-6.1.9.jar!/:6.1.9]
        at io.undertow.servlet.core.ManagedFilter.doFilter(ManagedFilter.java:67) ~[undertow-servlet-2.3.13.Final.jar!/:2.3.13.Final]
        at io.undertow.servlet.handlers.FilterHandler$FilterChainImpl.doFilter(FilterHandler.java:131) ~[undertow-servlet-2.3.13.Final.jar!/:2.3.13.Final]
        at lavalink.server.io.ResponseHeaderFilter.doFilterInternal(ResponseHeaderFilter.kt:17) ~[classes!/:4.0.7]
        at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116) ~[spring-web-6.1.9.jar!/:6.1.9]
        at io.undertow.servlet.core.ManagedFilter.doFilter(ManagedFilter.java:67) ~[undertow-servlet-2.3.13.Final.jar!/:2.3.13.Final]
        at io.undertow.servlet.handlers.FilterHandler$FilterChainImpl.doFilter(FilterHandler.java:131) ~[undertow-servlet-2.3.13.Final.jar!/:2.3.13.Final]
        at org.springframework.web.filter.CharacterEncodingFilter.doFilterInternal(CharacterEncodingFilter.java:201) ~[spring-web-6.1.9.jar!/:6.1.9]
        at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116) ~[spring-web-6.1.9.jar!/:6.1.9]
        at io.undertow.servlet.core.ManagedFilter.doFilter(ManagedFilter.java:67) ~[undertow-servlet-2.3.13.Final.jar!/:2.3.13.Final]
        at io.undertow.servlet.handlers.FilterHandler$FilterChainImpl.doFilter(FilterHandler.java:131) ~[undertow-servlet-2.3.13.Final.jar!/:2.3.13.Final]
        at io.undertow.servlet.handlers.FilterHandler.handleRequest(FilterHandler.java:84) ~[undertow-servlet-2.3.13.Final.jar!/:2.3.13.Final]
        at io.undertow.servlet.handlers.security.ServletSecurityRoleHandler.handleRequest(ServletSecurityRoleHandler.java:62) ~[undertow-servlet-2.3.13.Final.jar!/:2.3.13.Final]
        at io.undertow.servlet.handlers.ServletChain$1.handleRequest(ServletChain.java:68) ~[undertow-servlet-2.3.13.Final.jar!/:2.3.13.Final]
        at io.undertow.servlet.handlers.ServletDispatchingHandler.handleRequest(ServletDispatchingHandler.java:36) ~[undertow-servlet-2.3.13.Final.jar!/:2.3.13.Final]
        at io.undertow.servlet.handlers.RedirectDirHandler.handleRequest(RedirectDirHandler.java:68) ~[undertow-servlet-2.3.13.Final.jar!/:2.3.13.Final]
        at io.undertow.servlet.handlers.security.SSLInformationAssociationHandler.handleRequest(SSLInformationAssociationHandler.java:117) ~[undertow-servlet-2.3.13.Final.jar!/:2.3.13.Final]
        at io.undertow.servlet.handlers.security.ServletAuthenticationCallHandler.handleRequest(ServletAuthenticationCallHandler.java:57) ~[undertow-servlet-2.3.13.Final.jar!/:2.3.13.Final]
        at io.undertow.server.handlers.PredicateHandler.handleRequest(PredicateHandler.java:43) ~[undertow-core-2.3.13.Final.jar!/:2.3.13.Final]
        at io.undertow.security.handlers.AbstractConfidentialityHandler.handleRequest(AbstractConfidentialityHandler.java:46) ~[undertow-core-2.3.13.Final.jar!/:2.3.13.Final]
        at io.undertow.servlet.handlers.security.ServletConfidentialityConstraintHandler.handleRequest(ServletConfidentialityConstraintHandler.java:64) ~[undertow-servlet-2.3.13.Final.jar!/:2.3.13.Final]
        at io.undertow.security.handlers.AuthenticationMechanismsHandler.handleRequest(AuthenticationMechanismsHandler.java:60) ~[undertow-core-2.3.13.Final.jar!/:2.3.13.Final]
        at io.undertow.servlet.handlers.security.CachedAuthenticatedSessionHandler.handleRequest(CachedAuthenticatedSessionHandler.java:77) ~[undertow-servlet-2.3.13.Final.jar!/:2.3.13.Final]
        at io.undertow.security.handlers.AbstractSecurityContextAssociationHandler.handleRequest(AbstractSecurityContextAssociationHandler.java:43) ~[undertow-core-2.3.13.Final.jar!/:2.3.13.Final]
        at io.undertow.server.handlers.PredicateHandler.handleRequest(PredicateHandler.java:43) ~[undertow-core-2.3.13.Final.jar!/:2.3.13.Final]
        at io.undertow.servlet.handlers.SendErrorPageHandler.handleRequest(SendErrorPageHandler.java:52) ~[undertow-servlet-2.3.13.Final.jar!/:2.3.13.Final]
        at io.undertow.server.handlers.PredicateHandler.handleRequest(PredicateHandler.java:43) ~[undertow-core-2.3.13.Final.jar!/:2.3.13.Final]
        at io.undertow.servlet.handlers.ServletInitialHandler.handleFirstRequest(ServletInitialHandler.java:276) ~[undertow-servlet-2.3.13.Final.jar!/:2.3.13.Final]
        at io.undertow.servlet.handlers.ServletInitialHandler$2.call(ServletInitialHandler.java:135) ~[undertow-servlet-2.3.13.Final.jar!/:2.3.13.Final]
        at io.undertow.servlet.handlers.ServletInitialHandler$2.call(ServletInitialHandler.java:132) ~[undertow-servlet-2.3.13.Final.jar!/:2.3.13.Final]
        at io.undertow.servlet.core.ServletRequestContextThreadSetupAction$1.call(ServletRequestContextThreadSetupAction.java:48) ~[undertow-servlet-2.3.13.Final.jar!/:2.3.13.Final]
        at io.undertow.servlet.core.ContextClassLoaderSetupAction$1.call(ContextClassLoaderSetupAction.java:43) ~[undertow-servlet-2.3.13.Final.jar!/:2.3.13.Final]
        at io.undertow.servlet.handlers.ServletInitialHandler.dispatchRequest(ServletInitialHandler.java:256) ~[undertow-servlet-2.3.13.Final.jar!/:2.3.13.Final]
        at io.undertow.servlet.handlers.ServletInitialHandler$1.handleRequest(ServletInitialHandler.java:101) ~[undertow-servlet-2.3.13.Final.jar!/:2.3.13.Final]
        at io.undertow.server.Connectors.executeRootHandler(Connectors.java:393) ~[undertow-core-2.3.13.Final.jar!/:2.3.13.Final]
        at io.undertow.server.HttpServerExchange$1.run(HttpServerExchange.java:859) ~[undertow-core-2.3.13.Final.jar!/:2.3.13.Final]
        at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18) ~[jboss-threads-3.5.0.Final.jar!/:3.5.0.Final]
        at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2513) ~[jboss-threads-3.5.0.Final.jar!/:3.5.0.Final]
        at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1538) ~[jboss-threads-3.5.0.Final.jar!/:3.5.0.Final]
        at org.xnio.XnioWorker$WorkerThreadFactory$1$1.run(XnioWorker.java:1282) ~[xnio-api-3.8.8.Final.jar!/:3.8.8.Final]
        at java.base/java.lang.Thread.run(Thread.java:1575) ~[na:na]
        Suppressed: dev.lavalink.youtube.ClientInformation:
  yts.version: 1.11.1
  client.identifier: TVHTML5_SIMPLY_EMBEDDED_PLAYER
  client.options: ClientOptions{playback=true, playlistLoading=true, videoLoading=true, searching=true}
Caused by: java.lang.RuntimeException: TVHTML5_EMBEDDED cannot be used to load playlists
        ... 75 common frames omitted
```
### What is your client configuration?
```
    clients:       - "ANDROID_MUSIC"       - "MUSIC"       - "WEB"       - "WEBEMBEDDED"       - "TVHTML5EMBEDDED"     ANDROID_MUSIC:       playlistLoading: false       videoLoading: true       searching: true       playback: true     MUSIC:       playlistLoading: false       videoLoading: true       searching: true       playback: false     WEB:       playlistLoading: false       videoLoading: true       searching: true       playback: true     WEBEMBEDDED:       playlistLoading: false       videoLoading: false       searching: false       playback: true     TVHTML5EMBEDDED:       playlistLoading: false       videoLoading: false       searching: false       playback: true
```

**Comment by devoxin** [comment]
Provide debug logs.

**Comment by devoxin** [comment]
Fixed in ccd7cebf822cc5a423f78d26fff65f73a1c85a5b
